### ISSUE TITLE
Does this actually works ?

### ISSUE THREAD

**Issue by devkarlos** [kapilpipaliya/todo_drogon_server#3]
Hello,

In my project I'm working with websockets and I'm using drogon framework. I'm looking for some projects on github to get known how it works and I'm was trying get to work your project, but unsuccessfully.. I'm on windows and I noticed you are on Arch. I was trying to orient in your code but I can't. So I'm asking does this actually works ?

Thank you

**Comment by kapilpipaliya** [comment]
yes, it was working. this is some outdated project. the current updated project is not hosted on github.I dont use postgresql now. 

I am happy to help you on your project setup.

can we discuss what problems you have?

there is also drogon gitter channel where author is helping for all types of questions.

**Comment by kapilpipaliya** [comment]
I recommend you create new project from scratch.
https://github.com/drogonframework/drogon/wiki/ENG-03-Quick-Start

**Comment by devkarlos** [comment]
My project is working, but I was looking for some examples on github and your project interested me, because you are using websockets and I can learn the logic of it. But what your project have to do ? Its called todo, but it also looks like some music server or idk. Can you describe it ?

**Comment by kapilpipaliya** [comment]
Thank you for your interest. I am using caf framework in this project,
I was making a eCommerce website back-end with this project, then I also tried to make todo application like asana, i also tried to make a music server in same project. please check past two commit, i deleted extra files, but it may be useful to you.

**Comment by devkarlos** [comment]
Alright, thank you. I will try to get it to work and if I get into some troubles I will contact you.

**Comment by kapilpipaliya** [comment]
You can contact me at kapilpipaliya@gmail.com
